granulate_utils/metrics/spark.py

```diff
@@ -143,6 +150,11 @@ def _get_standalone_apps(self) -> Dict[str, Tuple[str, str]]:
                     pass
             except Exception:
                 self._logger.exception("Error was found while iterating applications.")
+        else:
+            if running_apps == {}:
+                self._logger.debug("running_apps is empty", metrics_json=metrics_json)
+            if current_running_apps == []:
+                self._logger.debug("current_running_apps is empty", metrics_json=metrics_json)
```

Comment:
Always log the size of running apps & current running apps, in a single log message.

Reply:
ee3d79a